src/MakeLoveNotWar.js

```diff
@@ -0,0 +1,11 @@
+var arr = []
+
+var increaseFunction = function increaseFunction(i) {
+  arr.push(function (x) { return x + i; });
+};
+
+for (var i = 0; i < 10; i++) {
```

Comment:
Maybe you know maybe not, just in case: if you don't really need that separate function, all you want to achieve is for JS to create separate context for you, it is OK to just create and immediately invoke the function.

Reply:
Do you mean like this? (in new commit)